### PR TITLE
Update get-assessment-methods-properties.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated properties  and added rbacGroupId and os version

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5877

File: microsoft-365/security/defender-endpoint/get-assessment-methods-properties.md [hunk 1]
```diff
@@ -69,20 +69,22 @@ Export secure configuration assessment **(via files)**|Secure configuration by d
...
-IsExpectedUserImpact|bool|Indicates whether there will be user impact if the configuration will be applied
-OSPlatform|string|Platform of the operating system running on the device. This indicates specific operating systems, including variations within the same family, such as Windows 10 and Windows 7. See tvm supported operating systems and platforms for details.
-RbacGroupName|string|The role-based access control (RBAC) group. If this device is not assigned to any RBAC group, the value will be "Unassigned." If the organization doesn't contain any RBAC groups, the value will be "None."
-RecommendationReference|string|A reference to the recommendation ID related to this software.
-Timestamp|string|Last time the configuration was seen on the device
+configurationCategory|string|Category or grouping to which the configuration belongs: Application, OS, Network, Accounts, Security controls
+configurationId|string|Unique identifier for a specific configuration
+configurationImpact|string|Rated impact of the configuration to the overall configuration score (1-10)
+configurationName|string|Display name of the configuration
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-assessment-methods-properties.md]:
```suggestion
configurationCategory|string|Category or grouping to which the configuration belongs: Application, OS, Network, Accounts, Security controls.
configurationId|string|Unique identifier for a specific configuration.
configurationImpact|string|Rated impact of the configuration to the overall configuration score (1-10).
configurationName|string|Display name of the configuration.
```

File: microsoft-365/security/defender-endpoint/get-assessment-methods-properties.md [hunk 2]
```diff
@@ -69,20 +69,22 @@ Export secure configuration assessment **(via files)**|Secure configuration by d
...
+configurationSubcategory|string|Subcategory or subgrouping to which the configuration belongs. In many cases, this describes specific capabilities or features.
+deviceId|string|Unique identifier for the device in the service.
+deviceName|string|Fully qualified domain name (FQDN) of the device.
+isApplicable|bool|Indicates whether the configuration or policy is applicable
+isCompliant|bool|Indicates whether the configuration or policy is properly configured
+isExpectedUserImpact|bool|Indicates whether there will be user impact if the configuration will be applied
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-assessment-methods-properties.md]:
```suggestion
isApplicable|bool|Indicates whether the configuration or policy is applicable.
isCompliant|bool|Indicates whether the configuration or policy is properly configured.
isExpectedUserImpact|bool|Indicates whether there will be user impact if the configuration will be applied.
```

File: microsoft-365/security/defender-endpoint/get-assessment-methods-properties.md [hunk 3]
```diff
@@ -69,20 +69,22 @@ Export secure configuration assessment **(via files)**|Secure configuration by d
...
+osPlatform|string|Platform of the operating system running on the device. This indicates specific operating systems, including variations within the same family, such as Windows 10 and Windows 7. See tvm supported operating systems and platforms for details.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-assessment-methods-properties.md]:
```suggestion
osPlatform|string|Platform of the operating system running on the device. This indicates specific operating systems, including variations within the same family, such as Windows 10 and Windows 7. See TVM supported operating systems and platforms for details.
```

File: microsoft-365/security/defender-endpoint/get-assessment-methods-properties.md [hunk 4]
```diff
@@ -69,20 +69,22 @@ Export secure configuration assessment **(via files)**|Secure configuration by d
...
+osVersion|string|Specific version of the operating system running on the device.
+rbacGroupName|string|The role-based access control (RBAC) group. If this device is not assigned to any RBAC group, the value will be "Unassigned." If the organization doesn't contain any RBAC groups, the value will be "None."
+rbacGroupId|string|The role-based access control (RBAC) group ID.
+recommendationReference|string|A reference to the recommendation ID related to this software.
+timestamp|string|Last time the configuration was seen on the device
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-assessment-methods-properties.md]:
```suggestion
timestamp|string|Last time the configuration was seen on the device.
```

File: microsoft-365/security/defender-endpoint/get-assessment-methods-properties.md [hunk 5]
```diff
@@ -111,10 +113,10 @@ DeviceName|string|Fully qualified domain name (FQDN) of the device.
 DiskPaths|Array[string]|Disk evidence that the product is installed on the device.
 EndOfSupportDate|string|The date in which support for this software has or will end.
 EndOfSupportStatus|string|End of support status. Can contain these possible values: None, EOS Version, Upcoming EOS Version, EOS Software, Upcoming EOS Software.
-Id|string|Unique identifier for the record.
 NumberOfWeaknesses|int|Number of weaknesses on this software on this device
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-assessment-methods-properties.md]:
```suggestion
NumberOfWeaknesses|int|Number of weaknesses on this software on this device.
```

File: microsoft-365/security/defender-endpoint/get-assessment-methods-properties.md [hunk 6]
```diff
@@ -155,6 +157,7 @@ Id|string|Unique identifier for the record.
 LastSeenTimestamp|string|Last time the CVE was seen on the device.
 OSPlatform|string|Platform of the operating system running on the device. This indicates specific operating systems, including variations within the same family, such as Windows 10 and Windows 7. See tvm supported operating systems and platforms for details.
 RbacGroupName|string|The role-based access control (RBAC) group. If this device is not assigned to any RBAC group, the value will be "Unassigned." If the organization doesn't contain any RBAC groups, the value will be "None."
+rbacGroupId|string|The role-based access control (RBAC) group ID.
 RecommendationReference|string|A reference to the recommendation ID related to this software.
 RecommendedSecurityUpdate|string|Name or description of the security update provided by the software vendor to address the vulnerability.
 RecommendedSecurityUpdateId|string|Identifier of the applicable security updates or identifier for the corresponding guidance or knowledge base (KB) articles
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-assessment-methods-properties.md]:
```suggestion
RecommendedSecurityUpdateId|string|Identifier of the applicable security updates or identifier for the corresponding guidance or knowledge base (KB) articles.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba Please copy review

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@jweston-1 Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@jweston-1 Gentle reminder

**Pull request merged**